docs/content/troubleshooting.md

```diff
@@ -1,113 +1,119 @@
 
-**Issue and workarounds**<br/>
+**Cause**<br/>
 
-* The struct you defined implements `IComponentData` and contains [non-blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) fields. Types that implement
+The struct you defined implements [`IComponentData`](https://github.com/Unity-Technologies/EntityComponentSystemSamples/blob/132f511a0f36d2bb422fc807cb3a808ea18d7df5/Documentation/content/ecs_in_detail.md#icomponentdata) and contains [non-blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) fields. Types that implement
 `IComponentData` may only contain blittable fields.
 
-* `bool` as well as `System.Boolean` are not blittable. Please use `Improbable.Gdk.Core.BlittableBool`, a blittable alternative
-implementation we provide, in that case.
+The most common causes of this exception are `bool` and `System.Boolean`, both of which are not blittable.
+
+**Fix**<br/>
+
+ Instead of `bool` and `System.Boolean`, please use `Improbable.Gdk.Core.BlittableBool`, a blittable alternative implementation we provide, in that case.
 
 <br/>
 
-### Exception thrown 
-When building a worker:<br/> 
-_`<path to burst package>\.Runtime\bcl.exe` did not run properly!_
+#### `Runtime\bcl.exe` did not run properly!
 
-**Issue and workarounds**<br/>
-This is a benign exception that is thrown when building a worker while burst compilation is turned on. Your worker was successfully built despite this error. To remove the error message, disable Burst compilation by unchecking `Jobs > Enable Burst Compilation`
-in the Unity Editor.
+**Cause**<br/>
 
-<br/>
+This is a benign exception that is thrown when building a worker while burst compilation is turned on. Your worker was successfully built despite this error.
 
-## Log warnings
+**Workaround**<br/>
 
-#### Unity Editor logs warning
+To remove the error message, disable Burst compilation by unchecking `Jobs > Enable Burst Compilation` in the Unity Editor.
 
-The Unity Editor logs a bunch of warnings when I open the SpatialOS GDK Unity project of the  for the first time. 
+## Warnings
 
-**Issue and workarounds**<br/>
-The warnings originate from code written by Unity, e.g. from `com.unity.mathematics` or `com.unity.entities`.
-These warnings are caused by code maintained by Unity over which we have no control. The warnings are benign so you can ignore them.
+#### Several warnings when opening the project for the first time
 
-<br/>
+**Cause**<br/>
+These warnings originate from code written by Unity, `com.unity.mathematics` or `com.unity.entities` for example. These warnings are benign so you can ignore them.
 
-#### Unity Editor logs warning
-Several error messages are logged after I destroyed a GameObject or ECS entity representing a SpatialOS entity.
+#### Several warnings when deleting a GameObject or ECS entity that represents a SpatialOS entity
```

Comment:
I'm surprised this only throws warnings

Reply:
I'm too busy to test this now so will leave this as is and double check what is thrown next time I deploy.